rules/unix-shell.data

```diff
@@ -51,6 +51,8 @@ bin/ar
 bin/arch
 bin/aria2c
 bin/arj
+bin/arjdisp
+bin/arj-register
```

Comment:
Necessary, because this file is used to generate the contents of `unix-shell-upto3.ra` and `unix-shell-4andup.ra`. In `unix-sehll-upto3.ra`, `arj` would be suffixed with `@`, hence `arjdisp` and `arj-register` would no longer be matched by rules using those two files.